docker/runme-build-env.Dockerfile

```diff
@@ -7,7 +7,6 @@ LABEL org.opencontainers.image.title="Runme build environment"
 LABEL org.opencontainers.image.description="An image to build and test runme."
 
 ENV HOME=/root
-ENV SHELL=/bin/bash
```

Comment:
Without this Dagger builds will fail. This might very well be an issue in the underyling container image. However, it deserves it's own PR to address this because without it will fail.